chihiro_todo/main.js

```diff
@@ -7,10 +7,18 @@ const doneArea = document.querySelector('#done_area');
 // taskをデータとして管理する変数
 let taskList = [];
 
+//担当者をデータとして管理する変数
+const personList = [{ id: "1", name: "柿崎"}, {id: "2", name: "柴田"}, {id: "3", name: "黒澤"}];
+
+//プルダウンで選択された担当者と一致するidをpersonListから探し出し、担当者名で返却する関数
+const getPersonNameById = (personId) => {
+    return personList.find(person => person.id === personId).name;
+}
+
 // タスクを追加する関数
-const addTask = (newTask) => {
+const addTask = function(newTask, selectPerson) {
```

Comment:
@chihiro-kakizaki 
`選択された担当者ID`という意味で`selectedPersonId`と命名してあげた方が適切かと思いました！
```suggestion
const addTask = function(newTask, selectedPersonId) {
```

Reply:
引数名の変更及び、renderSelectPersonBox関数内においてvalueの戻り値が不要だったため、iを削除しました。

Reply:
@chihiro-kakizaki 
ありがとう！
こんな感じで対応したことを箇条書きにすると、より見やすいと思います！

例: ======
以下のコミットで3点対応いたしました。
https://github.com/chihiro-kakizaki/todolist/pull/3/commits/cab86679c173a53c226560867027e1c1bde11dd4
- `selectPerson`という変数名を`selectedPersonId`に修正
- `renderSelectPersonBox`関数内の`map`関数の引数の`index`は不要だったため削除
- `querySelector`で`select`要素を取得する箇所を1箇所に集約
======
上の3つをそれぞれ別のコミットに分けてあげてもよかったかもね！

Githubのマークダウンの記法についても、知っておいた方が、より見やすい記述をできるから、積極的に使っていくようにしよう！
https://gist.github.com/mignonstyle/083c9e1651d7734f84c99b8cf49d57fa

Reply:
ご指摘ありがとうございます🙇
今後は修正点をコミットごとに分けて対応するように気をつけます。
また、マークダウン記法も意識して使うようにしたいと思います！！

Reply:
担当者の追加機能についても作成しましたので併せてご確認お願い致します🙇‍♀️